static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -492,31 +492,100 @@ $block-height: 58px;
 
 }
 
-.fc-item--type-review {
-    .fc-item__header {
-        .stars {
-            background-color: $news-garnett-highlight;
-            display: inline-block;
-            line-height: 0;
-            padding: calc((#{$gs-baseline} / 2) - 2px);
-            margin-top: $gs-baseline / 2;
+//Base stars styling on facia cards *****
+
+.fc-item__media-wrapper,
+.fc-item__header
+{
+    .fc-item__stars {
+        background-color: $news-garnett-highlight;
+        display: inline-block;
+        line-height: 0;
+        padding: $gs-baseline / 4;
+        margin-top: $gs-baseline / 6;
+    }
+
+    .inline-star {
+        display: inline-block;
+        width: .65em;
+
+        @include mq($from: desktop) {
+            width: 1em;
+        }
+
+        svg {
+            stroke: $garnett-neutral-1;
+            fill: transparent;
+            height: .75em;
+            width: .75em;
+
+            @include mq($from: desktop) {
+                height: 1em;
+                width: 1em;
+            }
+        }
+    }
+
+    .star__item--golden {
+        svg {
+            fill: $garnett-neutral-1;
+            stroke: none;
+        }
+    }
+}
+
+//Specific card type styling overides
+.fc-item__media-wrapper {
+    position: relative;
+    .fc-item__stars {
+        position: absolute;
+        z-index: 10;
+        bottom: 0;
+    }
+}
+
+.fc-item--list-media-tablet,
+.fc-item--list-mobile
+{
+    .inline-star {
+        width: .65em;
+
+        svg {
+            stroke: $garnett-neutral-1;
+            fill: transparent;
+            height: .75em;
+            width: .75em;
         }
+    }
+}
 
-        .inline-star {
-            display: inline;
+.fc-item--has-image {
+    .fc-item__header {
```

Comment:
it's it important that `fc-item__stars` sits below `fc-item__header` here? (np if so, if not, could we drop `fc-item__header`?)

Reply:
Yeah there are couple of scenarios where it's only the stars that sit within header that need to be hidden, and the stars that sit over the image will be visible. :-)